pyproject.toml

```diff
@@ -0,0 +1,87 @@
+[build-system]
+requires = ["setuptools>=63", "wheel", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "AxonDeepSeg"
+dynamic = ["version"]
+description = "Axon/Myelin segmentation using AI"
+readme = "README.md"
+requires-python = ">=3.11"
+license = {text = "MIT"}
+authors = [{"name" = "NeuroPoly Lab, Polytechnique Montreal", "email" = "axondeepseg@googlegroups.com"}
+]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: napari",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Scientific/Engineering :: Image Processing",
+]
+
+dependencies =[
+    "numpy<2",
+    "scipy",
+    "scikit-image",
```

Comment:
Diffing the [list of dependencies](https://www.diffchecker.com/cTaOYkdt/), I notice that there are some changes:

 - `napari` -> `napari[all]`
 - added `pytest-qt`, `magicgui`, and `qtpy`

It looks like some of these are specific to the Napari plugin (from `plugins/setup.cfg`). Would it make sense to remove the new dependencies from this PR, and then re-add them in the Napari-specific PR? (Just to show which dependencies got added from merging Napari into ADS?)

Of course this is like, pretty nitpicky and not 100% necessary, but it would create a  nice record of how the transition evolved between PRs.

Reply:
Good catch - yeah these are here too early =) Will remove

Reply:
Resolved in 096eac0

Reply:
It seems that removing `pytest-qt` caused the tests to fail:

```
file /Users/runner/work/axondeepseg/axondeepseg/ads_conda/envs/venv_ads/lib/python3.11/site-packages/napari/utils/_testsupport.py, line 178
  @pytest.fixture
  def make_napari_viewer(
E       fixture 'qtbot' not found
```

I'm surprised why this doesn't fail on the main branch?

Reply:
Sorry for all the hassle. Feel free to preserve these dependencies then merge as-is. I don't want to hold up the transition any longer than it needs to be due to organizational nitpicks. :sweat:

Reply:
Probably due to this line that's still in there that I had also removed,

[pytest-qt](https://github.com/axondeepseg/axondeepseg/blob/2334be33a5e4c3bd2f3bdbf459d710de7b067d60/requirements.txt#L25C1-L25C10)

I think you had maybe missed it when comparing, and I removed it since you mentionned it but didn't compare myself. So looks like that's the only very necessary one, but I'll keep the others since they're coming in the next PR anyways